### ISSUE TITLE
cannot change subs and dubs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tcitworld/android-vlc-remote#77]
```
What steps will reproduce the problem?
1.when i press the buttons with the mic and the letter 'a' on the interfCe, 
they fpash orange but nothing happens, though every other function is working 
fine
2.
3.

What is the expected output? What do you see instead?
The buttons just flash but nothing happens.

What version of VLC Remote are you using?
The latest beta version (i downloaded it 2 nights ago)

What version of VLC are you using?
Version 1.0.2 goldeneye (32 bit)

What version of Android are you using?
Version 2.2.1

Which Android device are you using?
Htc magic

Please provide any additional information below.

Basically what ive said above, every other button/toggle is working on the 
remote except the ones for changing the subtitles and audio tracks.

```

Original issue reported on code.google.com by `Naomi.G...@gmail.com` on 25 Apr 2011 at 8:42
- Merged into: #60
